packages/core/src/actions/smartAccount/sendTransactions.ts

```diff
@@ -31,20 +31,16 @@ export const sendTransactions: <
     );
   }
 
-  const { batch, overrides: _overrides } = await buildUserOperationFromTxs(
-    client,
-    {
-      requests,
-      overrides,
-      account,
-    }
-  );
-
-  const { hash } = await sendUserOperation(client, {
-    uo: batch,
-    overrides: _overrides,
+  const { uoStruct } = await buildUserOperationFromTxs(client, {
+    requests,
+    overrides,
     account,
   });
 
+  const { hash } = await _sendUserOperation(client, {
+    account: account as SmartContractAccount,
+    uoStruct,
+  });
```

Comment:
this is gonna run the middleware stack again, so you need to pass the overrides from above through to here.

I don't think `buildUserOperationFromTx` was ever meant to actually call `buildUserOperation`... which I admit makes the name super confusing

Reply:
this doesn't run middleware pipeline https://github.com/alchemyplatform/aa-sdk/blob/d08ac65a7e9d3ee1d82c803d25911804afdb4a05/packages/core/src/actions/smartAccount/internal/sendUserOperation.ts#L14 this is the internal send user op func

Reply:
oh wait you're right

Reply:
should we then update the `buildUserOperationFromTx` as well?

Reply:
oh it already does this... gotcha so it's just `buildUserOperationFromTxs` that's weird

Reply:
lol